library/src/main/java/in/krharsh17/barview/BarGroup.java

```diff
@@ -121,7 +122,7 @@ void setupBar() {
         ));
         bar.setPadding(0,dp(1), 0, dp(1));
         this.addView(bar);
-        bar.setBackgroundColor(Color.parseColor(color));
+        bar.setBackground(SetUpRoundBars(Color.parseColor(color)));
```

Comment:
Function names should begin with a lowercase letter.
```suggestion
        bar.setBackground(setUpRoundBars(Color.parseColor(color)));
```